core/Twig.php

```diff
@@ -360,6 +360,11 @@ protected function addFilter_notification()
     protected function addFilter_safeDecodeRaw()
     {
         $rawSafeDecoded = new TwigFilter('rawSafeDecoded', function ($string) {
+
+            if (!$string) {
```

Comment:
This one actually changes the result. The string `0` will no be returned as empty string. That can be seen in the UI test results. Maybe we need to check if it is a string

Reply:
@sgiehl PR has been updated